extensions/node-menu/renderer.tsx

```diff
@@ -1,21 +1,21 @@
 import { Registry, LensRendererExtension } from "@k8slens/extensions";
 import React from "react"
-import { NodeMenu } from "./src/node-menu"
+import { NodeMenu, NodeMenuProps } from "./src/node-menu"
 
 export default class NodeMenuRendererExtension extends LensRendererExtension {
   async onActivate() {
     console.log("node-menu extension activated")
   }
 
   registerKubeObjectMenus(registry: Registry.KubeObjectMenuRegistry) {
```

Comment:
`registry` is not used anymore, should we remove it?

Reply:
yes, removed